static/js/actions/email.js

```diff
@@ -33,13 +38,13 @@ export function sendSearchResultMail(
   searchRequest: Object
 ): Dispatcher<EmailSendResponse> {
   return (dispatch: Dispatch) => {
-    dispatch(initiateSendEmail());
+    dispatch(initiateSendEmail(SEARCH_EMAIL_TYPE));
     return api.sendSearchResultMail(subject, body, searchRequest).
       then(response => {
-        dispatch(sendEmailSuccess());
+        dispatch(sendEmailSuccess(SEARCH_EMAIL_TYPE));
         return Promise.resolve(response);
       }).catch(error => {
-        dispatch(sendEmailFailure(error));
+        dispatch(sendEmailFailure({type: SEARCH_EMAIL_TYPE, error: error}));
```

Comment:
yeah. when we implement something that allows the containers to reuse these functions, the `emailType` would be specified ahead of time and would replace this reference